stdlib/float.mli

```diff
@@ -253,6 +300,23 @@ val compare: t -> t -> int
 val equal: t -> t -> bool
 (** The equal function for floating-point numbers, compared using {!compare}. *)
 
+val min : t -> t -> t
```

Comment:
I disagree.  The symmetry of min is expected to hold and the generic `min` is not for floats because it expects a total order.  Actually using the asymmetry of `Stdlib.min` for float would be very much ill advised.  IMHO, one must choose whether ignoring nan (treating them as unavailable data) or returning nan as soon as there is one (treating them as “errors”).  I chose the former because that's what, for example, Julia does.

Reply:
To be sure there is no misunderstanding: you finally settled on introducing new names as suggested, right?